docs/components/board/_index.md

```diff
@@ -546,6 +546,71 @@ err := myBoard.WriteAnalog(context.Background(), "11", 48, nil)
 {{% /tab %}}
 {{< /tabs >}}
 
+### StreamTicks
+
+Start a stream of [`DigitalInterrupt`](/components/board/#digital_interrupts) ticks.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `interrupts` (List[[DigitalInterrupt](https://python.viam.dev/autoapi/viam/components/board/board/index.html#viam.components.board.board.Board.DigitalInterrupt)]): List of digital interrupts to receive ticks from.
+
+**Returns:**
+
+- [(`TickStream`)](https://python.viam.dev/autoapi/viam/components/board/board/index.html#viam.components.board.board.TickStream): [Stream](https://python.viam.dev/autoapi/viam/streams/index.html#viam.streams.Stream) of [Ticks](https://python.viam.dev/autoapi/viam/proto/component/board/index.html#viam.proto.component.board.StreamTicksResponse), objects containing `pin_name`, `time`, and `high` fields.
+
+```python {class="line-numbers linkable-line-numbers"}
+my_board = Board.from_robot(robot=robot, name="my_board")
+
+di8 = await my_board.digital_interrupt_by_name(name="8")
+di11 = await my_board.digital_interrupt_by_name(name="11")
+
+# Stream ticks from the listed digital interrupts on pins 8 and 11.
+ticks = await my_board.stream_ticks([di8, di11])
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/board/client/index.html#viam.components.board.client.BoardClient.stream_ticks).
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `interrupts` [([]DigitalInterrupt)](https://pkg.go.dev/go.viam.com/rdk/components/board#DigitalInterrupt): Slice of digital interrupts to receive ticks from.
+- `ch` ([chan](https://go.dev/tour/concurrency/2) [Tick](https://pkg.go.dev/go.viam.com/rdk/components/board#Tick)): The channel to stream Ticks, structs containing `Name`, `High`, and `TimestampNanosec` fields.
+- `extra` [(map\[string\]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/board#Board).
+
+```go
+myBoard, err := board.FromRobot(robot, "my_board")
+
+// Make a channel to stream ticks
+ticksChan := make(chan board.Tick)
+
+interrupts := []*DigitalInterrupt{}
+
+if di8, err := myBoard.DigitalInterruptByName("8"); err == nil {
+    interrupts = append(interrupts, di8)
+}
+if di11, err := myBoard.DigitalInterruptByName("11"); err == nil {
+    interrupts = append(interrupts, di11)
+}
+
+// Stream ticks on ticksChan from the listed digital interrupts on pins 8 and 11.
+err := myBoard.StreamTicks(context.Background(), interrupts, ticksChan, nil)
```

Comment:
@oliviamiller Is `err := ` incorrect here? I cribbed this partially from [adxl345 code](https://github.com/viamrobotics/rdk/blob/b1ad5688640d562a20282b297b3337fa6e2a2cbf/components/movementsensor/adxl345/adxl345.go#L341) and noticed it was `=` instead but I've never seen that with variable initialization so thought I'd double check

Reply:
Ok thank you!